demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -339,6 +342,51 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
             }
     }
     
+    public func credentialDisplayRendering(arguments: Dictionary<String, Any>, result: @escaping FlutterResult){
+   
+        
+        guard let resolvedCredentialDisplayData = arguments["resolvedCredentialDisplayData"] as? String else{
+            return  result(FlutterError.init(code: "NATIVE_ERR",
+                                             message: "error while credentialDisplayRendering",
+                                             details: "parameter resolvedCredentialDisplayData is missed"))
+        }
+        
+        let displayData = DisplayParseData(resolvedCredentialDisplayData, nil)
+        let issuerDisplayData = displayData?.issuerDisplay()
+        
+        var resolvedCredDisplayList : [Any] = []
+        var claimList:[Any] = []
+        
+        for i in 0...((displayData!.credentialDisplaysLength())-1){
+            let credentialDisplay = displayData!.credentialDisplay(at: i)
+            for i in 0...(credentialDisplay!.claimsLength())-1{
+                let claim = credentialDisplay?.claim(at: i)
```

Comment:
Consider simply force unwrapping `credentialDisplay`, as it will be never be nil as long as the loop logic is correct (in Wallet-SDK, I only return `nil` if you attempt to get an object at an out of bounds index)

Reply:
updated